### DISCUSSION TITLE
Pay 15% support for a partner to build a 100T miner.  I have no knoweledge.

### DISCUSSION THREAD

**Discussion by johnc16888** [filecoin-project/community#265] [Mining]
Hello,
I want to build a lotus miner with about 100 T storage. I only have some basic computer knowledge. I would like to know the price for a such filecoin miner and I would like to pay 15% of my production as compensation. I will pay all of the hardware and you help me to build it up and make it work.
If you are interested in helping me to build this miner, please contact me at tomanchenwin@gmail.com.

John

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
see your mails - happy to help.

**Comment by johnc16888** [comment]
I am very glad to hear from you.
I don’t know how much money to start building a miner, if it is more than
$8,000 US, I need to wait until next year March to get my funds ready.
It is too late tonight, I may respond your email tomorrow.

Thanks lot

John
